torch/utils/_content_store.py

```diff
@@ -53,7 +53,7 @@ def lazy_compile(**compile_kwargs):
     def decorate_fn(fn):
```

Comment:
It's probably worth updating the doc here too, since this isn't strictly a lazy compile only anymore; it also forces compilation (so it's really a wrapper oriented towards authoring eager kernels)